lib/smart_proxy_dns_plugin_template/dns_plugin_template_version.rb

```diff
@@ -1,7 +1,7 @@
 module Proxy
   module Dns
     module PluginTemplate
-      VERSION = '0.1'
+      VERSION = '0.2'
```

Comment:
Not sure this needs to be increased since most modules will start with a 0.1 version.